CHANGES.md

```diff
@@ -1,6 +1,8 @@
 # Changes
 
 ## Unreleased - 2021-xx-xx
+* Added TestServer::ws_at_with_config method
```

Comment:
standard changelog format:

```suggestion
### Added
* Added `TestServer::ws_at_with_config` method. [#2097]

```

...with PR referenced as others are. Similar change needed in other changelog.